### PR TITLE
Send ahrs groundspeed estimate instead of GPS groundspeed in VFR_HUD message

### PR DESCRIPTION
**Pull request by floaledm**
Discussed with @tridge today who said this would be a good idea to use ahrs groundspeed estimate instead of GPS groundspeed in VFR_HUD messages.

@tridge also said there should be some followup cleanup whereas instead of calling groundspeed_vector() throughout the code, the value(s) should be calculated during the ahrs update... I'll address that soon in another PR.

### CODE REVIEW

File: ArduPlane/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -443,7 +443,7 @@ void Plane::send_vfr_hud(mavlink_channel_t chan)
     mavlink_msg_vfr_hud_send(
         chan,
         aspeed,
-        gps.ground_speed(),
+        ahrs.groundspeed_vector().length(),
```
Comment (OXINARF) [inline ArduPlane/GCS_Mavlink.cpp]: There is a *ahrs.groundspeed()* function that does exactly this.

**Comment by OXINARF** [comment]
I think we should decide what all the vehicles send as right now they are inconsistent in what they do in the two speed fields:

- Plane: airspeed and GPS/groundspeed
- Copter: GPS and GPS speed
- Rover: GPS and groundspeed

I would say only Plane respects the MAVLink message. Not sure what to do with Rover and Copter since they don't have airspeed.

**Comment by floaledm** [comment]
@OXINARF Thanks for your suggestion on ahrs.groundspeed(). I made the change on both PRs.

I think you're right re. consistency. At the very least, the VFR_HUD groundspeed field for Copter should be ahrs.groundspeed() to be the same as in Plane and Rover (should I add it to this PR or create another?). Now, as to what should be sent for Copter and Rover for the airspeed field of VFR_HUD... what do others think?

**Comment by tridge** [comment]
in copter and rover we just send groundspeed for now. We may support airspeed sensors on copters and boats in the future, but not yet

**Pull request merged**